components/datadog/agent/ecsFargate.go

```diff
@@ -44,6 +44,10 @@ func ECSFargateLinuxContainerDefinition(e config.Env, image string, apiKeySSMPar
 				Name:  pulumi.StringPtr("DD_ECS_TASK_COLLECTION_ENABLED"),
 				Value: pulumi.StringPtr("true"),
 			},
+			ecs.TaskDefinitionKeyValuePairArgs{
+				Name:  pulumi.StringPtr("DD_PROCESS_CONFIG_PROCESS_COLLECTION_ENABLED"),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should this be enabled by default? Should it be allowed to be disabled? I guess we can do it through the agent env vars

Reply:
We'd want this to be enabled by default, for consistency with the ECS EC2 agent:
https://github.com/DataDog/test-infra-definitions/blob/7e55b9e3279af35d88e4965820114d00e659942b/components/datadog/agent/ecs.go#L134-L137

For some context, this came up when launching an ECS cluster with the invoke task and noticing that the ECS EC2 tasks in the cluster were having their processes collected, but the Fargate tasks in the cluster weren't